src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -255,4 +270,72 @@ private static void writeDelegates(final GameData data, final ObjectOutputStream
     out.writeObject(DELEGATE_LIST_END);
   }
 
+  /**
+   * Loads game data from the specified stream, which is expected to be in serializable format.
+   *
+   * @param is The stream from which the game data will be loaded; must not be {@code null}. The caller is responsible
+   *        for closing this stream; it will not be closed when this method returns.
+   *
+   * @return The loaded game data; never {@code null}.
+   *
+   * @throws IOException If an error occurs while loading the game.
+   */
+  public static GameData loadSerializableGame(final InputStream is) throws IOException {
```

Comment:
Good stuff with the load/save code in general for the rest of this class. I like that the abstractions are consistent, and take out some tedious details while staying at something of a high level. It's not necessarily easy to achieve that combination, but I've found it to be generally a useful structure that is also easy to test. :clinking_glasses: